### PR TITLE
Setup auth store

### PR DESCRIPTION
**Pull request by amyria3**

### CODE REVIEW

File: src/components/Button.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ interface ButtonProps {
   height: "default" | "small"
   label: string;
   onClick: (event: React.FormEvent<any>) => void;
-  disabled: boolean; //state
+  disabled?: true; //state
```
[the remark this answers is not in the document]
Comment (amyria3) [reply src/components/Button.tsx:11]: I dont really know how it somes in, i guess due to my patch? I dodn't stage and push this file. I was confused too.

File: src/stores/AuthStore.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+//Create a User type that has a username and a avatarUrl
+type User = {
+    username: string;
+    avatarUrl: string;
+}
+
+//Create a type for your AuthStore that can hold a user and a token, as well as updaters
+type AuthStore = {
+    token: string;
+    user: User
```
[the remark this answers is not in the document]
Comment (amyria3) [reply src/stores/AuthStore.ts:13]: thanx :)

[the remark this answers is not in the document]
Comment (amyria3) [reply src/stores/AuthStore.ts]: done

**Comment by amyria3** [comment]
How should I proceed here, @franzwollang ? :)

**Review by franzwollang — APPROVED** [review]

**Pull request merged**